README.md

```diff
@@ -1 +1,130 @@
-# renku-graph-vis
\ No newline at end of file
+# renku-graph-vis plugin
+
+## Graphical visualization of the graph
+Starting from the knowledge graph extracted from the renku project, this is queried to retrieve the needed information, 
```

Comment:
I suggest to feed it through GPT or @ferrigno. Or I will adapt it a bit later.

Reply:
Or I can keep making concrete suggestions for improvement, if you prefer.

Reply:
I did a first round of adaption using ChatGPT. In the meantime you can comment on a more overall level, is some thing missing? something should be removed/added?